vizro-core/examples/scratch_dev/pure-app.py

```diff
@@ -0,0 +1,66 @@
+"""Dev app to try things out."""
```

Comment:
I'll delete this file before merge.

Reply:
I've tested several combinations and they worked like a charm.